regression-test/suites/query_p0/join/test_bitmap_filter_nereids.groovy

```diff
@@ -0,0 +1,95 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_bitmap_filter_nereids") {
+    def tbl1 = "test_query_db.bigtable"
+    def tbl2 = "bitmap_table_nereids"
+    def tbl3 = "test_query_db.baseall"
+
+    sql "set runtime_filter_type = 16"
+
+    sql "DROP TABLE IF EXISTS ${tbl2}"
+    sql """
+    CREATE TABLE ${tbl2} (
+      `k1` int(11) NULL,
+      `k2` bitmap BITMAP_UNION NULL,
+      `k3` bitmap BITMAP_UNION NULL
+    ) ENGINE=OLAP
+    AGGREGATE KEY(`k1`)
+    COMMENT 'OLAP'
+    DISTRIBUTED BY HASH(`k1`) BUCKETS 2
+    PROPERTIES (
+    "replication_allocation" = "tag.location.default: 1"
+    );
+    """
+    sql """
+    insert into ${tbl2} values 
+    (1, bitmap_from_string('1, 3, 5, 7, 9, 11, 13, 99, 19910811, 20150402'),
+        bitmap_from_string('32767, 1985, 255, 789, 1991')),
+    (2, bitmap_from_string('10, 11, 12, 13, 14'), bitmap_empty());"""
+
+    sql "set enable_nereids_planner=true;"
+    sql "set enable_fallback_to_original_planner=false;"
+
+    qt_sql1 "select k1, k2 from ${tbl1} where k1 in (select k2 from ${tbl2}) order by k1;"
```

Comment:
add correlation subquery cases

Reply:
correlation subquery is not supported by "in (select bitmap...)"